operator/.gitignore

```diff
@@ -24,3 +24,5 @@ Dockerfile.cross
 *.swp
 *.swo
 *~
+
+operator.tar.gz
```

Comment:
This can be removed if we are already ignoring the `out/` folder

Reply:
Fixed in 7d80cb4.